src/Engine/Component/Physics.hpp

```diff
@@ -61,5 +60,14 @@ namespace Component {
              */
             glm::vec3 momentOfInertia = glm::vec3(1.f, 1.f, 1.f);
 
+        public:
+            /// Create new physics component.
+            /**
+            * @param entity Pointer to which entity this component corresponds.
+            */
```

Comment:
Because we want everything to be pretty, please insert spaces so that the * align!